### PR TITLE
fix #79092 revouninstaller.com

### LINKED ISSUE #79092: www.revouninstaller.com
### Issue URL (Ads)
 [https://www.revouninstaller.com/?x-clickref=1100lfswZXCp](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.revouninstaller.com%2F%3Fx-clickref%3D1100lfswZXCp)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist

### PR DESCRIPTION
**Pull request by wout-codes**
#79092

### CODE REVIEW

**Review by BlazDT — COMMENTED** [review]
Imho self-promo and should not be blocked. @AdguardTeam/filters-maintainers

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -98,6 +98,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+revouninstaller.com#$##popupModal { display: none !important; }
+revouninstaller.com#$#.modal-backdrop { display: none !important; }
+revouninstaller.com#$#body { overflow: visible !important; padding-right: 0 !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:103]: Зачем `padding-right: 0 !important;`?

Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt:103]: Yeah, the padding changes when the overflow is hidden.

Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt:103]: image

**Review by zloyden — APPROVED** [review]

**Pull request merged**